package.json

```diff
@@ -11,33 +11,31 @@
   },
   "homepage": "https://github.com/jest-community/jest-runner-eslint",
   "files": [
-    "build/"
+    "src/"
   ],
   "scripts": {
     "test": "jest",
-    "lint": "eslint .",
-    "watch": "babel src -w --ignore **/*.test.js,integration -d build",
-    "build": "babel src --ignore **/*.test.js,integration -d build",
-    "prepublish": "yarn build",
-    "format": "prettier --single-quote --trailing-comma all --write \"!(build)/**/*.js\""
+    "lint": "eslint . --config ./.eslintrc.json --no-eslintrc",
```

Comment:
had to do this to avoid loading https://github.com/jest-community/jest-runner-eslint/blob/master/integrationTests/__fixtures__/custom-parser/.eslintrc.json